src/components/alerts/ReviewGuidelinesAlert.tsx

```diff
@@ -0,0 +1,231 @@
+import {
+  Button,
+  Flex,
+  ListItem,
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+  UnorderedList,
+} from "@chakra-ui/react";
+import { useRouter } from "next/router";
+import { forwardRef, useRef } from "react";
+
+type Props = {
+  isOpen: boolean;
+  isFirstTime?: boolean;
+  onCancel: () => void;
+};
+
+type GuidelinesContentProps = Omit<Props, "isOpen">;
+
+const ReviewGuidelinesAlert = ({ isOpen, onCancel }: Props) => {
+  const cancelRef = useRef<HTMLButtonElement>(null);
+  const router = useRouter();
+  const isFirstTime = router.query.first_review === "true" ? true : false;
+  const handleClose = () => {
+    onCancel();
+    router.query.first_review = "false";
+  };
+
+  return (
+    <Modal
+      size={"2xl"}
+      isOpen={isOpen}
+      onClose={!isFirstTime ? handleClose : () => {}}
+    >
+      <ModalOverlay>
+        <ModalContent
+          maxH={"600px"}
+          p={0}
+          position={"relative"}
+          overflowY={"auto"}
+        >
+          <GuidelinesContent
+            onCancel={handleClose}
+            ref={cancelRef}
+            isFirstTime={isFirstTime}
+          />
+        </ModalContent>
+      </ModalOverlay>
+    </Modal>
+  );
+};
+
+export default ReviewGuidelinesAlert;
+
+const GuidelinesContent = forwardRef<HTMLButtonElement, GuidelinesContentProps>(
+  ({ onCancel, isFirstTime }, ref) => {
+    return (
+      <>
+        <ModalHeader px={5} fontSize="lg" fontWeight="bold">
+          Review Guidelines
+        </ModalHeader>
+        <ModalBody p={0}>
+          <Text fontSize={"sm"} px={5} pb={2}>
+            To ensure a shared quality of transcripts, please review and make
+            sure you meet the following guidelines:
+          </Text>
+          <Flex flexDir={"column"} px={5} gap={2}>
+            <Text>Transcription Style:</Text>
+            <UnorderedList pl={3} color="gray.700" fontSize="14px">
+              <ListItem>
+                <Text>
+                  Aim for an &quot;edited transcription &quot; style, preserving
+                  text meaning without paraphrasing.
+                </Text>
+              </ListItem>
+              <ListItem>
+                <Text>
+                  Omit stammering, filler words (&apos;like,&apos; &apos;you
+                  know&apos;), and unnecessary non-verbal communication.
+                </Text>
+              </ListItem>
+              <ListItem>
+                <Text>
+                  Maintain completeness while ensuring readability by
+                  eliminating mid-sentence rephrasings and non-essential lines.
+                </Text>
+              </ListItem>
+            </UnorderedList>
+          </Flex>
+
+          <Flex my={2} px={5} flexDir={"column"} gap={2}>
+            <Text>Transcript Structure:</Text>
```

Comment:
Some of those headings have ":" and others do not. I think it's better without, so just "Transcript Structure".